frontend/cypress/integration/featureFiles/workload_logs.feature

```diff
@@ -0,0 +1,43 @@
+Feature: Workload logs tab
+  The Logs tab of a specific workload allows to see the generated logs of
+  its associated pods. If the workload is backed by more than one pod, the user
+  can choose which pod to see its logs. If the pod has multiple containers, it
+  is possible to view logs from all containers in a single view.
+
+  Background:
+    Given I open Kiali URL
+    Given user is at administrator perspective
+
+  Scenario: The logs tab should show the logs of a pod
+    Given I am on the "productpage-v1" workload detail page of the "bookinfo" namespace
+    When I go to the Logs tab of the workload detail page
```

Comment:
"Given" ... "And" ...

Reply:
I think here the wording is fine. As you say, the "Given" followed by "When" makes sense.